src/Mod/OpenSCAD/OpenSCADFeatures.py

```diff
@@ -235,7 +235,6 @@ def updateProperty(self, fp, prop, value):
             setattr(fp, prop, value)
 
     def execute(self, fp):
-        pass
```

Comment:
Removing this `pass` statement brings back dead code into life. Is this intensional and did you test the undead code? (I don't see any comments in either PR or commit message about it)

Reply:
That is not my understanding of how `pass` works -- of course, I've been wrong before! But `pass` is not `return`, it's just used as a placeholder where Python requires there to be some code and you don't plan on putting any (yet). So an empty function that you haven't written yet, etc. In this case the `pass` both literally and figuratively does nothing, and is not necessary.

Reply:
I'm so sorry  😓 You are of cause correct.
I wonder what the reason the original coder had for having it there then 🤔

Reply:
Reading it in its context made it clear that it must just be a stray `pass`